src/Uno.Toolkit.RuntimeTests/Tests/NavigationBarTests.cs

```diff
@@ -391,17 +429,18 @@ public NavBarTestPage()
 #endif
 	}
 
-#if __IOS__
+#if __IOS__ || __ANDROID__
 	public static class NavigationBarTestHelper
 	{
+#if __IOS__
 		public static UINavigationBar? GetNativeNavBar(this NavigationBar? navBar) => navBar
```

Comment:
I think it would be good to have the following from GetNativeNavBar indented the same way as GetNativeNavItem

`?.TryGetRenderer<NavigationBar, NavigationBarRenderer>()` 
`?.Native;`